src/GraphQL.Tests/Bugs/Issue2275.cs

```diff
@@ -71,7 +71,7 @@ public FilterInputGraphType()
         }
     }
 
-    [GraphQLMetadata(InputType = typeof(FilterInputGraphType))]
+    [InputType(typeof(FilterInputGraphType))]
```

Comment:
Please update the documentation to use these attributes

Reply:
I don't see any reference to the removed `InputType` property within the documentation -- ??

Reply:
Oh right, my mistake!